test-cypress/cypress/e2e/2-advanced-examples/cypress_api.cy.js

```diff
@@ -0,0 +1,184 @@
+      expect(myConfig).to.have.property('responseTimeout', 30000);
+      expect(myConfig).to.have.property('viewportHeight', 660);
+      expect(myConfig).to.have.property('viewportWidth', 1000);
+      expect(myConfig).to.have.property('pageLoadTimeout', 60000);
+      expect(myConfig).to.have.property('waitForAnimations', true);
+
+      expect(Cypress.config('pageLoadTimeout')).to.eq(60000);
+
+      // this will change the config for the rest of your tests!
+      Cypress.config('pageLoadTimeout', 20000);
+
+      expect(Cypress.config('pageLoadTimeout')).to.eq(20000);
+
+      Cypress.config('pageLoadTimeout', 60000);
+    });
+  });
+
+  context('Cypress.dom', () => {
+    beforeEach(() => {
+      cy.visit('https://example.cypress.io/cypress-api');
+    });
+
+    // https://on.cypress.io/dom
+    it('.isHidden() - determine if a DOM element is hidden', () => {
+      const hiddenP = Cypress.$('.dom-p p.hidden').get(0);
+      const visibleP = Cypress.$('.dom-p p.visible').get(0);
+
+      // our first paragraph has css class 'hidden'
+      expect(Cypress.dom.isHidden(hiddenP)).to.be.true;
+      expect(Cypress.dom.isHidden(visibleP)).to.be.false;
+    });
+  });
+
+  context('Cypress.env()', () => {
+    beforeEach(() => {
+      cy.visit('https://example.cypress.io/cypress-api');
+    });
+
+    // We can set environment variables for highly dynamic values
+
+    // https://on.cypress.io/environment-variables
+    it('Get environment variables', () => {
+    // https://on.cypress.io/env
+    // set multiple environment variables
+      Cypress.env({
+        host: 'veronica.dev.local',
+        api_server: 'http://localhost:8888/v1/'
+      });
+
+      // get environment variable
+      expect(Cypress.env('host')).to.eq('veronica.dev.local');
+
+      // set environment variable
+      Cypress.env('api_server', 'http://localhost:8888/v2/');
+      expect(Cypress.env('api_server')).to.eq('http://localhost:8888/v2/');
+
+      // get all environment variable
+      expect(Cypress.env()).to.have.property('host', 'veronica.dev.local');
+      expect(Cypress.env()).to.have.property('api_server', 'http://localhost:8888/v2/');
+    });
+  });
+
+  context('Cypress.log', () => {
+    beforeEach(() => {
+      cy.visit('https://example.cypress.io/cypress-api');
+    });
+
+    it('Control what is printed to the Command Log', () => {
+    // https://on.cypress.io/cypress-log
+    });
+  });
+
+  context('Cypress.platform', () => {
+    beforeEach(() => {
+      cy.visit('https://example.cypress.io/cypress-api');
+    });
+
+    it('Get underlying OS name', () => {
+    // https://on.cypress.io/platform
+      expect(Cypress.platform).to.be.exist;
+    });
+  });
+
+  context('Cypress.version', () => {
+    beforeEach(() => {
+      cy.visit('https://example.cypress.io/cypress-api');
+    });
+
+    it('Get current version of Cypress being run', () => {
+    // https://on.cypress.io/version
+      expect(Cypress.version).to.be.exist;
```

Comment:
There is a minor typo in the assertion. Instead of `to.be.exist`, it should be `to.exist` to correctly assert the existence of `Cypress.version`.